pmd-modelica/src/main/java/net/sourceforge/pmd/lang/modelica/ast/ASTStoredDefinition.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.modelica.ast;
+
+import net.sourceforge.pmd.lang.modelica.resolver.CompositeName;
+
+/**
+ * A representation of a Modelica source code file.
+ */
+public class ASTStoredDefinition extends AbstractModelicaNode {
```

Comment:
This seems to be the root node of the AST. Please mark this by implementing the marker interface `net.sourceforge.pmd.lang.ast.RootNode`.